client/src/Tools/_framework/Paths/ActivityEditor.tsx

```diff
@@ -365,28 +371,41 @@ export function ActivityEditor() {
         availableFeatures={availableFeatures}
         displayTab={displaySettingsTab}
       />
-      <ShareDrawer
-        isOpen={sharingIsOpen}
-        onClose={sharingOnClose}
-        finalFocusRef={controlsBtnRef}
-        fetcher={fetcher}
-        contentData={activityData}
-        allLicenses={allLicenses}
-        currentDoenetML={textEditorDoenetML}
-      />
-      <AssignmentSettingsDrawer
-        isOpen={assignmentSettingsAreOpen}
-        onClose={assignmentSettingsOnClose}
-        finalFocusRef={controlsBtnRef}
-        fetcher={fetcher}
-        id={activityId}
-        contentData={activityData}
-      />
-      <AssignmentInvitation
-        isOpen={invitationIsOpen}
-        onClose={invitationOnClose}
-        activityData={activityData}
-      />
+
+      {activityData.libraryActivity ? (
```

Comment:
A comment such as, if `libraryActivity` is present, it means the activity is the curated version in the library.

Reply:
Or better, since this is used multiple places, define a variable

```
const isLibraryActivity = Boolean(activityData.libraryActivity)
```